### PR TITLE
Record additional large text tests

### PR DESCRIPTION
**Pull request by dusi**
# 📲 What

Records large text tests for all size classes.

# 🤔 Why

I've recently added a test helper that lists all size class sizes using `allCases` accessor and didn't want to include it in the feature work so this is a separate PR.

# ✅ Acceptance criteria

- [ ] Tests pass
- [ ] Snapshots look legit

### CODE REVIEW

**Comment by Scollaco** [comment]
The snapshots are cutting out part of the switcher. Do you know what may be causing this?
Screenshot 2019-09-23 at 10 28 49

**Comment by dusi** [comment]
I believe that’s not cut off - it’s right at the edge. Github’s green border does not help here. Should check the raw image locally

**Comment by dusi** [comment]
I take that back...looks like it's cut off...will investigate

**Comment by dusi** [comment]
Should be fixed...looks like `rightAnchor` on `UISwitch` works in mysterious ways (see this playground)

Screen Shot 2019-09-23 at 10 18 56 AM

I've instead aligned toggle's `leftAnchor` to superview's `rightAnchor` minus the width of the toggle 😅 

Thanks for the attention to detail on this nino! 🔍

File: Kickstarter-iOS/Views/Controllers/ToggleViewController.swift [hunk 1]
```diff
@@ -30,12 +30,13 @@ class ToggleViewController: UIViewController {
...
     NSLayoutConstraint.activate([
+      self.view.heightAnchor.constraint(greaterThanOrEqualTo: self.toggle.heightAnchor),
       self.titleLabel.leftAnchor.constraint(equalTo: self.view.leftAnchor),
       self.titleLabel.topAnchor.constraint(equalTo: self.view.topAnchor),
       self.titleLabel.bottomAnchor.constraint(equalTo: self.view.bottomAnchor),
       self.titleLabel.centerYAnchor.constraint(equalTo: self.toggle.centerYAnchor),
-      self.titleLabel.rightAnchor.constraint(lessThanOrEqualTo: self.toggle.leftAnchor),
-      self.toggle.rightAnchor.constraint(equalTo: self.view.rightAnchor)
+      self.titleLabel.rightAnchor.constraint(equalTo: self.toggle.leftAnchor, constant: -Styles.grid(3)),
+      self.toggle.leftAnchor.constraint(equalTo: self.view.rightAnchor, constant: -toggle.frame.size.width)
```
Comment (Scollaco) [inline Kickstarter-iOS/Views/Controllers/ToggleViewController.swift]: The `-toggle.frame.size.width` is missing the `self.` here :)

**Pull request merged**